specs/DisableCrashReporting.md

```diff
@@ -0,0 +1,94 @@
+Process Info
+===
+
+# Background
+Currently, the crash process is created asynchronously with the WebView2 process and listening for any crashes. Once a crash happens, the crash pad process will create a crash dump and send that to Microsoft services for diagnosis. Provide end developer a new API to toggle dump upload consent, once uploads are disabled, dump will still be generated but just not uploaded to Microsoft endpoints. Also, provide an API to get the crash dump folder.
+
+# Examples
+## WinRT and .NET   
+```c#
+
+/// Create WebView Environment with option
```

Comment:
This sample is close to a real end to end scenario demonstration. If an end dev wants to do custom crash reporting, they should disable our crash reporting and then when there's a crash, get the files out of the crash dump folder and do something with them. We don't have to show doing something with the crash dumps (we can call a function like `ProcessNewCrashDumps(crashDumpFolder);` that we don't define) but we should show how they should know when to find new crash dumps. Can they use the ProcessFailed event - do we guarantee that the crash dumps will be totally written to disk by the time the ProcessFailed event is raised? Or do they have to do polling or some other way of watching for changes to the dump folder?

Reply:
The most reliable way for end developer is to monitor crash dump folder's file changes. (which is what DDG do)

Reply:
I verified using debugger. Dumps will get write to disk when ProcessFailed are raised. I updated the sample code section